Adds a learning bootcamp curriculum page (#1143)
BitcoinDesign/Guide: guide/resources/learning-bootcamp.md

---
layout: guide
title: Learning bootcamp
description: Take-home bitcoin design challenges to practice and play with.
nav_order: 4
parent: Resources
permalink: /guide/resources/learning-bootcamp/
main_classes: -no-top-padding
image: https://bitcoin.design/assets/images/guide/resources/learning-bootcamp/learning-bootcamp-preview.jpg
---

<!--

Editor's notes

This is learning material for a 4-week bootcamp, initially created for Summer of Bitcoin.
Challenge materials are not included, but can be requested.

-->

{% include picture.html
   image = "/assets/images/guide/resources/learning-bootcamp/learning-bootcamp.jpg"
   retina = "/assets/images/guide/resources/learning-bootcamp/learning-bootcamp@2x.jpg"
   mobile = "/assets/images/guide/resources/learning-bootcamp/learning-bootcamp-mobile.jpg"
   mobileRetina = "/assets/images/guide/resources/learning-bootcamp/learning-bootcamp-mobile@2x.jpg"
   alt-text = ""
   width = 1600
   height = 450
   layout = "full-width"
%}

[]({{ '' | relative_url }})

> # Bitcoin Design Learning Bootcamp

Reviewer: ```suggestion
# Self-Taught Bitcoin Design Bootcamp
```
My suggestion is to add a "Self-Taught" vibe to it. I feel, and I've observed, people are proud conveying that something they are good at, was something they taught themselves. I asked AI for suggestions too:

1. Bitcoin Design: Teach Yourself
2. The Self-Led Bitcoin Design Track
3. Bitcoin Design, The Self-Taught Way

I kind of like this 3rd one. But please choose on your discretion. Learning Bootcamp, sounds okay too !!

Edit: Saw your text, since it's mostly for SoB people having "Design Track" or "Learning Bootcamp" as the name makes sense !!

{:.no_toc}

> This is a four-week curriculum for self-guided learning. Initially created for [Summer of Bitcoin](http://summerofbitcoin.org), you may also go through the material independently, or use it for your own educational initiatives.

Reviewer: ```suggestion
Design is at its best when it makes the complex feel simple. That’s what Bitcoin Design is all about — taking this revolutionary technology and making it intuitive for the masses. If you want to contribute to this vision, this four-week self-paced curriculum will take you through the fundamentals of Bitcoin design, and help you build a strong foundation through engaging, real-world design challenges.

Originally created for [Summer of Bitcoin](http://summerofbitcoin.org), this track is now open for anyone who wants to learn independently or use it to run their own educational initiatives.
```

> Over the next four weeks, you'll explore the fundamentals of Bitcoin design — starting with the basics and progressing to more advanced wallet interactions. This program is designed to help you build a solid foundation while working on practical design challenges.

Reviewer: ```suggestion
Each week focuses on a key aspect of Bitcoin design, with curated reading materials to build your knowledge with a hands-on challenge:
```

- [Week 1: Bitcoin Design Fundamentals](#week-1-bitcoin-design-fundamentals)
- [Week 2: User Research & Personal Finance](#week-2-user-research--personal-finance)
- [Week 3: Keys, Wallet Creation, Backup](#week-3-keys-wallet-creation-backup)
- [Week 4: Wallet Design Patterns](#week-4-wallet-design-patterns)

> Each week, you’ll go through **2-3 hours of reading** followed by a **4-5 hour hands-on challenge**. The goal isn’t perfection—it’s learning by doing. Focus on working through the full design process within the given time rather than over-polishing. Submit what you can in those 5 hours, and remember: done is better than perfect.

Reviewer: ```suggestion
Expect about 2-3 hours of reading per week, followed by a 4-5 hour hands-on design challenge.

- **If you’re part of Summer of Bitcoin**, aim to complete the challenge within the given time frame — focus on working through the full design process rather than over-polishing. Submit what you can in those 5 hours, and remember: done is better than perfect.
- **If you're learning independently**, you can take your time, but we still encourage you to prioritize progress over perfection.
```

{% include tip/open.html color="blue" label="Note" icon="info" %}

> The challenges are not publicly included in this page. Since they are used as part of educational programs, we share them directly by request. If you are interested, reach out in Discord in the [#education](https://discord.gg/a2KzKc7BYk) channel.

Reviewer: ```suggestion
All reading materials are open to everyone. However, design challenges, **for independent learners**, are shared on request — just ask for access in the [#education](https://discord.gg/a2KzKc7BYk) channel on Discord when you're ready to tackle them.
```

{% include tip/close.html %}

Now let’s dive in!

### Week 1: Bitcoin Design Fundamentals

This week, we’re starting with the core principles of Bitcoin design and the first challenge: **Wallet Analysis**.

#### Learning Goals

By the end of this week, you should be able to:

* Understand core Bitcoin design principles and terminology  
* Learn open-source design practices and collaboration  
* Gain hands-on experience with different types of Bitcoin wallets  
* Develop critical analysis skills for wallet interfaces

#### Required Reading

These external resources will help you answer the reflection questions and complete the challenge.

Bitcoin Design Guide:
- [Getting Started]({{ '/guide/getting-started/' | relative_url }}) (Read all 8 pages)

Open Design Guide:
- [Open Design]({{ '/guide/open-design/' | relative_url }})
- [Understanding the Designer's Role]({{ '/guide/open-design/#the-designers-role' | relative_url }})

> #### Test Your Knowledge

Reviewer: ```suggestion
#### Test Your Understanding
```

---

## Week 2: User Research & Personal Finance

This week, you'll explore how people use Bitcoin in their financial lives and sharpen your research skills. You'll analyze how different Bitcoin wallets position themselves, identify user needs, and connect them to the personas you'll study.

#### Learning Goals

By the end of this week, you should be able to:

- Learn about Bitcoin user research methodologies  
- Develop secondary research skills — both finding and analyzing data  
- Understand user personas and how they apply to Bitcoin products  
- Get familiar with existing Bitcoin UX research

#### Required Reading

Bitcoin Design Guide / Designing bitcoin products:

- [Personal Finance]({{ '/guide/designing-products/personal-finance/' | relative_url }})  
- [Conducting Research]({{ '/guide/designing-products/user-research/' | relative_url }})  
- [User Personas]({{ '/guide/designing-products/user-personas/' | relative_url }})  
- [Getting to know your users]({{ '/guide/designing-products/getting-to-know-your-users/' | relative_url }})  

[UX Research Playbook](https://bitcoinresearch.xyz/playbook) (how to set up a research process)

> #### Test Your Knowledge

Reviewer: ```suggestion
#### Test Your Understanding
```

---

## Week 3: Keys, Wallet Creation, Backup

Understanding private key management is crucial for Bitcoin wallet design. This week focuses on creating intuitive interfaces for complex security concepts.

#### Learning Goals

By the end of this week, you should be able to:

- Learn private key management concepts and their importance in Bitcoin security  
- Understand the tradeoffs between security and usability in wallet design  
- Explore backup and recovery best practices  
- Develop intuitive user flows for complex security concepts

#### Required Reading

Bitcoin Design Guide:

- How it works / [Private Key Management]({{ '/guide/how-it-works/private-key-management/introduction/' | relative_url }}) 
- How it works / [Bitcoin Backups]({{ '/guide/how-it-works/backups/' | relative_url }})
- Daily spending wallet / [Wallet Creation]({{ '/guide/daily-spending-wallet/first-use/#creating-a-wallet' | relative_url }})
- Daily spending wallet / [Backup & Recovery]({{ '/guide/daily-spending-wallet/backup-and-recovery/landing-page/' | relative_url }})
- [Multiple Wallets Reference Design]({{ '/guide/multiple-wallets/' | relative_url }})

> #### Test Your Knowledge

Reviewer: ```suggestion
#### Test Your Understanding
```

The final week focuses on key wallet design patterns that accommodate various user needs and flows. You’ll explore how Bitcoin wallets handle transactions, requests, and privacy while ensuring usability and accessibility.

#### Learning Goals

By the end of this week, you should be able to:

- Understand core wallet interaction patterns and how they shape user experiences  
- Learn different payment request formats and their impact on transaction flows  
- Explore Lightning Network UX considerations and how they differ from on-chain payments  
- Design user-friendly, complex payment flows while maintaining security and efficiency

#### Required Reading

Bitcoin Design Guide:

- [Savings Wallet Reference Design]({{ '/guide/savings-wallet/' | relative_url }})  
- How it works / [Payment Request Formats]({{ '/guide/how-it-works/payment-request-formats/' | relative_url }})  
- How it works / [Lightning Services]({{ '/guide/how-it-works/lightning-services/' | relative_url }})  
- How it works / [Transactions]({{ '/guide/how-it-works/transactions/' | relative_url }})  
- How it works / [Wallet Privacy]({{ '/guide/how-it-works/wallet-privacy/' | relative_url }})

> #### Test Your Knowledge

Reviewer: ```suggestion
#### Test Your Understanding
```

- In the transaction diagram showing inputs and outputs, why does the sender need to combine both the 3 BTC and 2 BTC inputs to send 4 BTC? What happens to the extra 1 BTC?
- Why might a merchant require more than 1 confirmation before considering a transaction final? What risks are they protecting against?

**[Wallet Privacy]({{ '/guide/how-it-works/wallet-privacy/' | relative_url }})**

- What's the key difference between anonymity and pseudonymity in Bitcoin transactions? Why does this matter for user privacy?
- The page emphasizes generating new addresses for each payment. What specific privacy risks does address reuse create for both senders and receivers?
- How can transaction labeling and contact management help users maintain better privacy when using Bitcoin? What additional benefits does this provide?

#### Challenge: Design Details

This week, you’ll focus on refining wallet interactions to make Bitcoin more intuitive and user-friendly. Choose one of the challenges and design an interface that balances human experience with technical constraints. High-fidelity mockups are key here — your design should be polished, functional, and human-centered.

---

> Did you make it all the way through? Are you stuck and have questions? Join us on Discord in the [#education](https://discord.gg/a2KzKc7BYk) channel and share your journey. We're excited to chat with you.

Reviewer: ```suggestion
**You made it!** Four weeks of learning, thinking, and designing for Bitcoin — whether you completed everything or just explored parts of it, you’ve taken a solid step toward understanding Bitcoin design.

But this is just the beginning. The best way to keep learning is by applying your knowledge — contribute to open-source Bitcoin projects, keep iterating on your designs, and stay engaged with the community.

If you have questions, need feedback, or just want to share your work, join us in the **#education** channel on Discord. We’d love to see what you’ve built!
```
